modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -112,11 +112,11 @@ spec:
 <2> Specify the frequency that the kubelet checks the status of a node associated with this `MachineConfig` object. The default value is `10s`. If you change this default, the `node-status-report-frequency` value is changed to the same value.
 <3> Specify the frequency that the kubelet reports the status of a node associated with this `MachineConfig` object. The default value is `1m`.
 
-This parameter works with the `node-monitor-grace-period` and the `pod-eviction-timeout` parameters, which are not configurable. 
+The `node-status-update-frequency` parameter works with the `node-monitor-grace-period` and `pod-eviction-timeout` parameters.
 
-* The `node-monitor-grace-period` parameter specifies how long the {product-title} waits after a node associated with a `MachineConfig` object is marked `Unhealthy` if the controller manager does not receive the node heartbeat. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after the `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`. The `node-status-update-frequency` value must be lower than the `node-monitor-grace-period` value.
+* The `node-monitor-grace-period` parameter specifies how long {product-title} waits after a node associated with a `MachineConfig` object is marked `Unhealthy` if the controller manager does not receive the node heartbeat. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`. The `node-status-update-frequency` value must be lower than the `node-monitor-grace-period` value. Modifying the `node-monitor-grace-period` interval is not supported.
```

Comment:
What do you think about adding "the" back in front of `node-monitor-grace-period` for readability?

> If the remote worker node rejoins the cluster after _the_ `node-monitor-grace-period` expires, pods continue to run.

Reply:
Hi, @lbarbeevargas! Thanks for the comments!

I moved the no-support lines into the note as you suggested. That definitely looks better.

Not so sure about adding a definite article before an object name, but I can't find anywhere in our guides that discusses this specifically. It's an interesting case.

I think a good rule of thumb is to treat an object name as a proper noun, like Chris, Mississippi, or, say, rk86sh5.

Consider:

(+) Assign a value to the `chris` parameter.
(+) Assign a value to `chris`. 
(-) Assign a value to the `chris`.*
\* _'The' is clearly out of place here._

(+) Assign a value to the `rk86sh5` parameter.
(+) Assign a value to `rk86sh5`. 
(-) Assign a value to the `rk86sh5`.*
\* _'The' is clearly out of place here, too._

(+) Assign a value to the `node-monitor-grace-period` parameter.
(+?) Assign a value to `node-monitor-grace-period`.
(-?) Assign a value to the `node-monitor-grace-period`.
_The name of this parameter sounds like a normal phrase, so we're tuned in to hearing correct syntax. It sounds better to hear the definite article. (In English, at least: localized versions will treat this just like rk86sh5.)_

Does that explanation make sense?

Reply:
Thank you for the explanations on including or not including a definitive article to these examples. Sounds good to keep them as is! 👍